rocky/rocky/settings.py

```diff
@@ -69,24 +71,45 @@ class SaferExceptionReporterFilter(SafeExceptionReporterFilter):
 LOGGING = {
     "version": 1,
     "disable_existing_loggers": False,
-    "formatters": {
-        "json_formatter": {"()": structlog.stdlib.ProcessorFormatter, "processor": structlog.processors.JSONRenderer()},
-        "plain_console": {
-            "()": structlog.stdlib.ProcessorFormatter,
-            "processor": structlog.dev.ConsoleRenderer(
-                colors=True, pad_level=False, exception_formatter=structlog.dev.plain_traceback
-            ),
-        },
-    },
-    "handlers": {
-        "console": {
-            "class": "logging.StreamHandler",
-            "formatter": ("json_formatter" if LOGGING_FORMAT == "json" else "plain_console"),
-        }
-    },
+    "formatters": {"default": {"format": "%(message)s"}},
+    "handlers": {"console": {"class": "logging.StreamHandler", "formatter": "default"}},
     "loggers": {"root": {"handlers": ["console"], "level": env("LOG_LEVEL", default="INFO").upper()}},
 }
 
+
+def configure_logging(logging_settings):
+    log_cfg = env("ROCKY_LOG_CFG", default="")
```

Comment:
You can also use `Env.path`. Doesn't do validation, but does return a `Path` object. See [docs](https://django-environ.readthedocs.io/en/latest/api.html#environ.Env.path)

Reply:
I've changed it, but it would have been more useful if it returned a pathlib.Path object instead of an environ.Path object.